nfsstat/tests/compose/compose.yaml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+
+services:
+  nfs-server:
+    image: gists/nfs-server
```

Comment:
(nit) This one seems better supported with more releases: https://hub.docker.com/r/itsthenetwork/nfs-server-alpine/tags

Ideally we should use official images, but there isn't always one...